packages/reference/src/components/CreateEntryLinkButton/CreateEntryMenuTrigger.tsx

```diff
@@ -108,6 +109,10 @@ export const CreateEntryMenuTrigger = ({
 
   const closeMenu = () => setOpen(false);
 
+  useEffect(() => {
+    textField?.current?.querySelector('input')?.focus()
```

Comment:
The `querySelector` is a workaround as the `TextInput` component currently is a functional component without `forwardRef`, so we couldn't attach the ref directly on it. 😞

Reply:
wouldn't it focus and select all text every time? why this bug occurs at the first place? 

could you also leave a comment that this is a temporary workaround?

Reply:
It seems like whenever the `onChange` happens, the input field is rendered again as a new component and loses focus.
As for the `focus`, it keeps the last state of the element, so it should even keep the caret on the same place, it would select the entire text if it was using `select()`

Will also add this on the comment, to keep it clear why this useEffect is there too.

Reply:
can we use the autofocus property on the input element instead?

Reply:
I did try it with the `autofocus`, but it didn't work and also gives an eslint error, as autofocus is not recommended because of a11y